python/src/LinearTaylor_doc.i.in

```diff
@@ -7,41 +7,27 @@ Available constructors:
 Parameters
 ----------
 center : sequence of float
-    Point :math:`\vect{x}_0` where the Taylor expansion of the function
-    :math:`h` is performed.
+    Point :math:`\ux_0`.
 function : :class:`~openturns.Function`
-    Function :math:`h` to be approximated.
+    Function :math:`h` to be approximated around :math:`\ux_0`.
 
 Notes
 -----
-The approximation of the model response :math:`\vect{y} = h(\vect{x})` around a
-specific set :math:`\vect{x}_0 = (x_{0,1},\dots,x_{0,n_{X}})` of input
-parameters may be of interest. One may then substitute :math:`h` for its Taylor
-expansion at point :math:`\vect{x}_0`. Hence :math:`h` is replaced with a first
-or second-order polynomial :math:`\widehat{h}` whose evaluation is inexpensive,
-allowing the analyst to apply the uncertainty anaysis methods.
+We denote by :math:`h: \Rset^d \rightarrow \Rset^q` the function to be approximated and :math:`\uy=h(\ux)`.
```

Comment:
As an afterthought, I guess that refering directly to the _taylor_expansion help page would clarify the doc. It makes it possible to avoid duplicating the doc.

Reply:
yes